sdk_experimental/test/opentelemetry/sdk/trace/propagation/trace_context/response_text_map_propagator_test.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+
+describe OpenTelemetry::Trace::Propagation::TraceContext::ResponseTextMapPropagator do
+  Span = OpenTelemetry::Trace::Span
+  SpanContext = OpenTelemetry::Trace::SpanContext
+
+  let(:traceresponse_key) { 'traceresponse' }
+  let(:propagator) do
+    OpenTelemetry::Trace::Propagation::TraceContext::ResponseTextMapPropagator.new
+  end
+  let(:valid_traceresponse_header) do
+    '00-000000000000000000000000000000AA-00000000000000ea-01'
+  end
+  let(:invalid_traceresponse_header) do
+    'FF-000000000000000000000000000000AA-00000000000000ea-01'
+  end
+  let(:carrier) do
+    {
+      traceresponse_key => valid_traceresponse_header
+    }
+  end
+  let(:context) { OpenTelemetry::Context.empty }
+  let(:context_with_tracestate) do
```

Comment:
I don't see `tracestate` in here, but maybe I'm confused

Reply:
No it's not used anywhere, I'll rename this variable